teps/0044-decouple-task-composition-from-scheduling.md

```diff
@@ -860,55 +995,71 @@ Pros:
 Cons:
 * The line between this and a Pipeline seems very thin
 * New CRD to contend with
+* Will mean that both the Pipelines controller and this controller will need to share (or duplicate) a lot of
```

Comment:
How is this a con ? we share a lot of logic with knative, it ends up being in a library (`knative.dev/pkg`)